bucket/glimpse.json

```diff
@@ -0,0 +1,76 @@
+{
+    "version": "0.2.0",
+    "description": "Glimpse is an open source image editor based on the GNU Image Manipulation Program. The goal is to experiment with new ideas and expand the use of free software.",
+    "homepage": "https://glimpse-editor.github.io/",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "extract_dir": "PFiles\\Glimpse Image Editor\\Glimpse 0.2.0 (64-bit)",
+            "url": "https://github.com/glimpse-editor/Glimpse/releases/download/v0.2.0/glimpse-0.2.0-x64.msi",
+            "hash": "420dbbe11fef2a1d741083eb72f99732b9aec620d1d8274f9fb7a30e650030ff"
+        },
+        "32bit": {
+            "extract_dir": "PFiles\\Glimpse Image Editor\\Glimpse 0.2.0",
+            "url": "https://github.com/glimpse-editor/Glimpse/releases/download/v0.2.0/glimpse-0.2.0-i686.msi",
+            "hash": "2163403ca13d27be399b7fc1348719b2a9a4ddc4bc8c4b00aa8edd608bacfdbd"
+        }
+    },
+    "installer": {
+        "script": [
+            "Push-Location \"$dir\"",
+            "if ($architecture -eq '64bit') {",
+            "   $defpath = \"`nPATH=`${gimp_installation_dir}\\bin;`${gimp_installation_dir}\\32\\bin`n\"",
+            "} else {",
+            "   $defpath = \"`nPATH=`${gimp_installation_dir}\\bin`n\"",
+            "}",
+            "$defenv = Get-Content -Raw 'lib\\glimpse\\2.0\\environ\\default.env'",
+            "$defenv += $defpath",
+            "$defenv | Set-Content 'lib\\glimpse\\2.0\\environ\\default.env'",
+            "Pop-Location"
```

Comment:
```powershell
$defpath = 'PATH=${gimp_installation_dir}\\bin'
if ($architecture -eq '64bit') { $defpath = "$defpath;`${gimp_installation_dir}\\32\\bin" }
($defpath, (Get-Content -Raw "$dir\lib\..\default.env)) -join "`n""
Set-Content "$dir\lib\...default.env" -Encoding Ascii
```

Reply:
This cleanup should also be applied to `bucket/gimp.json`.